src/fastga/models/aerodynamics/external/propeller_code/compute_propeller_aero.py

```diff
@@ -55,31 +56,36 @@ def initialize(self):
             desc="Fineness between two blade pitch for the construction of the propeller efficiency"
             "map, in deg",
         )
+        self.options.declare("use_neuralfoil", default=False, types=bool)
 
     def setup(self):
         ivc = om.IndepVarComp()
         ivc.add_output("data:aerodynamics:propeller:mach", val=0.0)
         ivc.add_output("data:aerodynamics:propeller:reynolds", val=1e6)
         self.add_subsystem("propeller_efficiency_aero_conditions", ivc, promotes=["*"])
         for profile in self.options["sections_profile_name_list"]:
+            airfoil_polar = NeuralfoilPolar if self.options["use_neuralfoil"] else XfoilPolar
```

Comment:
If it works it's great, although we don't do it like this in other so I'm wondering it I should let this go